bench/micro/Bench/Database/LSMTree/Monoidal.hs

```diff
@@ -67,37 +64,39 @@ benchConfig = Common.defaultTableConfig {
 -- contents of the resulting database are the same.
 benchInsertsVsMupserts :: Benchmark
 benchInsertsVsMupserts =
-    env (pure $ snd $ randomEntriesGrouped 800_000 250) $ \ess -> bgroup "inserts-vs-mupserts" [
-        env (pure $ V.map mkNormalInserts ess) $ \inss -> bench "normal-inserts" $
-          withEmptyNormalTable $ \(_, _, _, _, t) ->
-            V.mapM_ (Normal.inserts t) inss
-      , env (pure $ V.map mkMonoidalInserts ess) $ \inss -> bench "monoidal-inserts" $
-          withEmptyMonoidalTable $ \(_, _, _, _, t) ->
-            V.mapM_ (Monoidal.inserts t) inss
-      , env (pure $ V.map mkMonoidalMupserts ess) $ \mupss -> bench "monoidal-mupserts" $
-          withEmptyMonoidalTable $ \(_, _, _, _, t) ->
-            V.mapM_ (Monoidal.mupserts t) mupss
-      ]
+    env (pure $ snd $ randomEntriesGrouped 800_000 250) $ \ess ->
+      env (pure $ V.map mkInserts ess) $ \inss ->
+        bgroup "inserts-vs-mupserts" [
+          bench "normal-inserts" $
+            withEmptyNormalTable $ \(_, _, _, _, t) ->
+              V.mapM_ (inserts t) inss
+        , bench "monoidal-inserts" $
+            withEmptyMonoidalTable $ \(_, _, _, _, t) ->
+              V.mapM_ (inserts t) inss
```

Comment:
At this point, there isn't really a difference between monoidal and normal inserts anymore, so we can probably just remove one of the two

Reply:
Done in 23e23ff8c1d7d9acc710a0acbab8b2e2a32f34bb.